fuse/CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.16)
 project(fuse)
-find_package(catkin REQUIRED)
-catkin_metapackage()
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
```

Comment:
Nit, no need to set the CXX standard since there's no code.

Reply:
https://github.com/locusrobotics/fuse/compare/c864d04d6b652b949a6b1ec70ec2f162706d5d01..b883f0f83486385a841bbc85b0a37a1b894e3304 !